architecture/opp-architecture.adoc

```diff
@@ -5,6 +5,39 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Discover {product-title} architecture and concepts.
+{product-title} is a single hybrid-cloud platform for enterprises to build, deploy, run, and manage intelligent applications securely across infrastructures. It is based on Red Hat Enterprise Linux, Kubernetes, and Red Hat {ocp}, and includes the following products:
 
-include::modules/opp-architecture-overview.adoc[leveloffset=+1]
\ No newline at end of file
+* Red Hat Advanced Cluster Management for Kubernetes - Controls clusters and applications from a single console.
+* Red Hat Advanced Cluster Security for Kubernetes - Provides information about cluster security, visibility management, and security compliance.
+* Red Hat Quay - A private container registry that stores, builds, and deploys container images.
+* Red Hat OpenShift Data Foundation Essentials - A data and storage services platform that provides a permanent place to store data while clusters spin up and down across environments.
+
+
+include::modules/opp-architecture-relnotes.adoc[leveloffset=+1]
+include::modules/opp-architecture-architecture.adoc[leveloffset=+1]
+include::modules/opp-architecture-compatibility-matrix.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../https://docs.openshift.com/container-platform/4.10/release_notes/ocp-4-10-release-notes.html[OpenShift Container Platform 4.10 release notes]
+* xref:../https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.0/html/release_notes/red-hat-advanced-cluster-management-for-kubernetes-release-notes[Red Hat Advanced Cluster Management for Kubernetes release notes]
+* xref:../https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/red_hat_quay_release_notes/index[Red Hat Quay Release Notes]
+* xref:../https://docs.openshift.com/acs/release_notes/369-release-notes.html[Red Hat Advanced Cluster Security for Kubernetes]
+* xref:../https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/4.9_release_notes/index[Red Hat OpenShift Data Foundation 4.9 release notes]
+
+include::modules/opp-architecture-installation.adoc[leveloffset=+1]
+include::modules/opp-architecture-support.adoc[leveloffset=+1]
+
+
+[role="_additional-resources"]
+[id="additional-resources"]
+== Additional resources
+.Installation
```

Comment:
Because these are all links, please incorporate them directly into the modules instead.